subsys/mgmt/fmfu/src/fmfu_mgmt.c

```diff
@@ -267,16 +278,13 @@ static struct mgmt_group fmfu_mgmt_group = {
 	.mg_group_id = (MGMT_GROUP_ID_PERUSER + 1),
 };
 
-int fmfu_mgmt_init(void)
+int fmfu_mgmt_register_group(fmfu_callback_t client_callback)
```

Comment:
IMO we should stick with the `init` name since the user really does not care what happenes in the background here (they don't even know what a "group" is in this context as they are not exposed to it.

Reply:
the idea here is to unify the API names. Since `os_mgmt_register_group` is a function and `fs_mgmt_register_group` etc. So it would make sense to have the same naming convention across all files registering handlers to MCUMgr